phpBB/includes/functions_compatibility.php

```diff
@@ -881,3 +881,65 @@ function phpbb_check_and_display_sql_report(\phpbb\request\request_interface $re
 
 	$controller_helper->display_sql_report();
 }
+
+/**
+ * Parse cfg file
+ * @param string $filename
+ * @param bool|array $lines
+ * @return array
+ *
+ * @deprecated 4.0.0-a1 (To be removed: 5.0.0)
```

Comment:
5.0.0?

Reply:
Could also be removed in 4.1

Reply:
I think is better tu put it deprecated in 3.3 (or 3.4) and remove it in 4.0

Reply:
Well it imho should be deprecated in the next feature or major version updated.